Article/Article.js

```diff
@@ -7,17 +7,45 @@ class Headline {
     
     // create a reference to the ".expandButton" class. 
     this.expandButton = this.domElement.querySelector('.expandButton');
+    this.redX = this.domElement.querySelector('i');
     
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    this.expandButton.textContent = 'expand'
+    this.expandButton.textContent = 'Click to Expand'
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
     this.expandButton.addEventListener('click', () => this.expandArticle())
+
+    this.redX.addEventListener('click', () => this.closeBox())
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
     this.domElement.classList.toggle('article-open')
+    if(this.domElement.classList.contains('article-open')) {
+      TweenMax.to(this.domElement, .8, {
+        height: 400,    
+        ease: Power4.easeOut,
+      });
+      this.expandButton.textContent = 'Click to Close'
+    }
+    else {
+      TweenMax.to(this.domElement, .8, {
+        height: 40,    
+        ease: Power4.easeOut,
+      });
+      this.expandButton.textContent = 'Click to Expand'
+    }
+   
+  }
+
+  closeBox() {
+    // this.domElement.style.display = 'none'
+    TweenMax.to(this.domElement, .5, {
+      scale: 0,    
+      rotation: 180,
+    });
+    // this.domElement.style.display = 'none'
```

Comment:
there is a way to make a function call after your tweenMax animation finishes. that function could be used to call the style.display = none; which would fix the visual bug of having huge blank spaces in the page where your articles used to live.